### PR TITLE
[Keyboard] Add Quarkeys Z60, reorganized keyboard file directory 

### PR DESCRIPTION
**Pull request by tommyamoszhao**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by waffle87** [comment]
Please stop closing and opening new PRs with the same content.

File: keyboards/quarkeys/z60/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Quarkeys Z60 Keyboard
+
+![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)
+![Layout](https://static.wixstatic.com/media/72b918_bb37ec16543545b0aac331d5a7fa977c~mv2.png)
```
Comment (waffle87) [inline keyboards/quarkeys/z60/readme.md:4]: This image needs to be rehosted on Imgur.
```suggestion
![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)
```

File: keyboards/quarkeys/z60/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Make example for this keyboard hotswap version keymap:
+
+    make quarkeys/z60/hotswap:default
+
+Flashing example for this keyboard:
+
+    make quarkeys/z60/solder:default:flash
+
+**Bootloader:** Press the physical reset button (ECS position of layer 1)
```
Comment (waffle87) [inline keyboards/quarkeys/z60/readme.md:25]:
```suggestion
**Bootloader:** Press the `QK_BOOT` keycode at ESC position of layer 1.
```

File: keyboards/quarkeys/z60/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                  Hotswap version with per-key RGBs, layer & Capslock indicators
+
+Make example for this keyboard solder version keymap(after setting up your build environment):
+
+    make quarkeys/z60/solder:default
+
+Make example for this keyboard hotswap version keymap:
+
+    make quarkeys/z60/hotswap:default
+
```
Comment (waffle87) [inline keyboards/quarkeys/z60/readme.md:20]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/quarkeys/z60/readme.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Quarkeys Z60 Keyboard
+
+![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)
```
Comment (drashna) [inline keyboards/quarkeys/z60/readme.md:3]: Please rehost on Imgur.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/quarkeys/z60/readme.md (hunk 4 above)
Comment (tommyamoszhao) [reply keyboards/quarkeys/z60/readme.md:3]: Hi drasma, I tried with Imgur but it has restriction for my region's cell number, so I guess I just delete the Picture in readme.md?

Comment (tommyamoszhao) [reply keyboards/quarkeys/z60/readme.md]: Hi, probably stupid to ask, do I need to click the 'Resolve conversation' button?

File: keyboards/quarkeys/z60/hotswap/Makefile [hunk 5]
```diff
@@ -0,0 +1,3 @@
+ifndef MAKEFILE_INCLUDED
```
Comment (waffle87) [inline keyboards/quarkeys/z60/hotswap/Makefile:1]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/quarkeys/z60/hotswap/Makefile [hunk 6]
```diff
@@ -1,3 +1 @@
-ifndef MAKEFILE_INCLUDED
-	include ../../Makefile
-endif
+include ../../Makefile
```
Comment (waffle87) [inline keyboards/quarkeys/z60/hotswap/Makefile:1]: > This file should be removed.

The **entire** file needs to be deleted. Not just the line I commented on.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/quarkeys/z60/hotswap/Makefile (hunk 6 above)
Comment (tommyamoszhao) [reply keyboards/quarkeys/z60/hotswap/Makefile]: ok, done

**Comment by tommyamoszhao** [comment]
Hi I'm sorry to bother, just want to ask how do this pr work, seems everything is done according to requests, is there anything else I need to do? or just wait quietly. Normally, how long it takes to merge successfully? This is my first pr attempt, appreciate, TommyZ

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**